packages/react/src/components/PageTitleBar/_page-title-bar.scss

```diff
@@ -72,10 +71,10 @@
     }
 
     .#{$prefix}--btn--secondary + .#{$prefix}--btn--primary {
-      margin-left: 0;
+      margin-left: 1px;
```

Comment:
I am surprised stylelint didn't bark at you for the use of `px`

Reply:
Yeah, I don't really understand how stylelint rules are applied sometimes.... 🤔